scripts/tests/e3/rules.lua

```diff
@@ -789,3 +928,36 @@ function test_volcanooutbreak_message()
     assert_not_equal("", msg:render("de"))
     assert_not_equal("", msg:render("en"))
 end
+
+
+function test_bug2083()
+    local herb_multi = 500 -- from rc_herb_trade()
+    local r = region.create(0,0,"plain")
+    r:set_resource("peasant", 2000) 
+    r.luxury = "balm"
+
+    local f = faction.create("2083@eressea.de", "human", "de")
+    local u = unit.create(f, r, 1)
+    u:set_skill("building", 8)
+    u:add_item("stone", 100)
+    u:add_item("log", 100)
+    u:add_item("iron", 100)
+    u:add_item("money", 10000)
+    u:clear_orders()
+    u:add_order("MACHE Markt")
+    process_orders()
+
+    -- this is a bit weird, but the bug was caused by market code
+    -- being called in two places. We want to make sure this doesn't happen
+    for k, v in pairs(rules) do
+        set_key("xm09", true)
```

Comment:
In der Schleife müsste es eigentlich nicht sein, aber wenn ich das nicht setze geistert unten, wo xmas2009.update aufgerufen wird, "es weihachtet sehr" über die Konsole, was ein bisschen irritierend (wenn auch nicht falsch!) ist. Wo ich drüber nachdenke, wäre es besser, dieses print dort zu entfernen oder das Modul ganz raus zu nehmen, denn gebraucht wird es nicht mehr, oder?